src/helpers/findAll.ts

```diff
@@ -0,0 +1,27 @@
+import { ReactTestInstance } from 'react-test-renderer';
+import { getConfig } from '../config';
+import { AccessibilityOption } from '../queries/accessibilityOption';
+import { isInaccessible } from './accessiblity';
+
+export function findAll<Options extends AccessibilityOption>(
```

Comment:
Do we need generics here, seem like it should work fine with concrete `AccessibilityOption` type.
```suggestion
export function findAll<Options extends AccessibilityOption>(
```

Reply:
agreed, dont remembrer why i did that